Explorer/Assets/DCL/MapRenderer/MapLayers/HomeMarker/HomeMarkerData.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using DCL.Diagnostics;
+using UnityEngine;
+
+namespace DCL.MapRenderer.MapLayers.HomeMarker
+{
+	public struct HomeMarkerData
```

Comment:
Im not sure of the value of this wrapper. Why does it exist, and why cant you just use `string`? (Or Vector2, if you decide to do a Vector2 serializer in `IDCLPrefs`)